### PR TITLE
Hide bundle tooltip for Joja members

### PR DESCRIPTION
**Pull request by Moonwolf287**
Added check if Jojo Member or if all bundles are done before building bundle info

Fix for #9

### CODE REVIEW

File: SDVModTest/UIElements/ShowItemHoverInformation.cs [hunk 1]
```diff
@@ -102,44 +102,47 @@ private void OnInventoryChanged(object sender, InventoryChangedEventArgs e)
...
-                            source.Add(bundleValue);
-                        }
-                    }
-
-                    if (source.Count > 0)
-                        _prunedRequiredBundles.Add(bundleName, source);
-                }
+	            foreach (var bundle in _bundleData)
+	            {
+		            String[] bundleRoomInfo = bundle.Key.Split('/');
```
Comment (cdaragorn) [inline SDVModTest/UIElements/ShowItemHoverInformation.cs]: Your tabbing seems to be set to 8. Please keep it at the 4 my project is using.

Comment (Moonwolf287) [reply SDVModTest/UIElements/ShowItemHoverInformation.cs]: Hm weird my usual default is 4 but I will look into it in a bit.

Comment (Moonwolf287) [reply SDVModTest/UIElements/ShowItemHoverInformation.cs]: Okay the problem is I did not replace tabs with spaces and github displays tabs as 8 spaces. (See: https://github.com/github/VisualStudio/issues/1524)
I will go through all my changes and replace all tabs with spaces.

**Comment by Zamiell** [comment]
This PR has been integrated in Annosz's fork: https://github.com/Annosz/UIInfoSuite2

**Pull request merged**